src/modules/vmount/output_mavlink.cpp

```diff
@@ -66,6 +66,9 @@ int OutputMavlinkV1::update(const ControlData *control_data)
 		//got new command
 		_set_angle_setpoints(control_data);
 
+#if !defined(ALT_MANTIS_GIMBAL_HACKS)
+		// Don't send this command to ATL Mantis as it just spams the vehicle_command queue and the
```

Comment:
Why is it spamming in the first place? Shouldn't `MAV_CMD_DO_MOUNT_CONFIGURE` only be spent on change in the first place? 

I obviously don't like the `ALT_MANTIS_GIMBAL_HACKS`, but it seems like something problematic outside of this situation we can improve slightly.

Reply:
Good point. I'll fix this and only send it if the control type changes.